README.md

```diff
@@ -23,6 +23,10 @@ After installation completes visit [127.0.0.1:3030](http://127.0.0.1:3030/). Use
 
 **Use `npm run update` to [fetch the latest changes](_scripts/update_all.sh) to servers**
 
+Note 1 : if npm fails when installing (specifically the fxa-content-server) or a number of servers fail to start, it may be worth trying a `npm install --production` and then a `npm install` to see if a two stage install works. Also, a `ulimit -n 2560` may help (to allow more currently open files) if you're on Linux.
+
```

Comment:
Yeah, @vladikoff told me this one. Funnily enough, I thought this was the whole reason izs wrote https://www.npmjs.com/package/graceful-fs ... perhaps we're doing it wrong! :D